manifests/plugin/ansible/runner.pp

```diff
@@ -0,0 +1,48 @@
+# = Ansible runner package installation
+#
+# $manage_runner_repo:: If true, adds upstream repositories to install ansible-runner package from
+#
+class foreman_proxy::plugin::ansible::runner(
+  Boolean $manage_runner_repo = $foreman_proxy::plugin::ansible::manage_runner_repo,
+) {
+
+  if $manage_runner_repo {
+    case $::osfamily {
```

Comment:
Let's start using the modern facts (we still need to convert the rest of the module):
```suggestion
    case $facts['os']['family'] {
```